src/test/java/com/revature/services/test/QuestionServiceTest.java

```diff
@@ -42,10 +47,12 @@
 	@MockBean
 	RSSService rssService;
 
+	private Date ts = new Date(100000);
+	
 	/** @author Hugh Thornhill */
 	@Test
 	public void getAllQuestionsTest() throws Exception {
-		Question question = new Question(1, 0, "Title", "Content", LocalDate.MIN, null, false, 1);
+		Question question = new Question(1, 0, "Title", "Content", ts, null, false, 1);
```

Comment:
We should not be using `java.sql.Date` unless you are conducting integration tests with the Database that necessitate using it. Use LocalDate for all internal application date handling.

Reply:
fixed